server.js

```diff
@@ -45,9 +45,9 @@ const didyouknowResponses = [
   'did you know chris is a framework slut',
   'did you know chris cried once watching an ad that popped up in youtube',
   'did you know chris used ipfs to avoid all his server cost problems',
-  'did you know chris has only one chrome extention and its his',
+  'did you know chris has only one chrome extention which he created',
```

Comment:
I know this isn't related to your changes here, but could we also get "extension" instead of extention.